src/unity/python/turicreate/test/test_logistic_classifier.py

```diff
@@ -355,7 +355,7 @@ def test__list_fields(self):
         """
         model = self.model
         fields = model._list_fields()
-        self.assertEqual(set(fields), set(self.fields_ans))
+        self.assertTrue(any(elem in set(fields) for elem in set(self.fields_ans)))
```

Comment:
I'm not sure this is the right fix. With this change, the behavior will only fail the test if none of the fields listed are in the expected set (if even one is, it seems it will pass, if I'm reading it correctly).

Instead, I think we should use the set methods to test for a subset. If `fields_ans` is a subset of (or same set as) `fields`, the test should pass. See the [docs](https://docs.python.org/2/library/sets.html#set-objects) for subset methods.

Reply:
We should also make sure we aren't breaking compatibility by adding a new field (which is ostensibly what this test is supposed to be testing):

* Does loading an old model (saved before this field existed) still work and/or give expected results?
* Does saving a new model (after this field exists) still work when you load it in an older version of Turi Create?